src/widgets/copilot/components/Messages/OutgoingMsg.tsx

```
@@ -2,46 +2,98 @@
+    let mapUrl;
+    if (latitude && longitude) {
+      mapUrl = buildOpenStreetMapEmbedUrl(latitude, longitude);
+    } else {
+      input_prompt ||= button_pressed?.button_title || "";
+    }
+
+    return (
+      <div className="gooey-outgoingMsg gmb-12 gpl-16">
+        {input_images.length > 0 &&
+          input_images.map((url: string) => (
+            <a href={url} target="_blank">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)